cmd/driver/install/install.go

```diff
@@ -136,17 +119,13 @@ func (o *driverInstallOptions) RunDriverInstall(ctx context.Context) (string, er
 		return "", nil
 	}
 
-	d, err := driverdistro.Discover(kr, o.Driver.HostRoot)
-	if err != nil {
-		if errors.Is(err, driverdistro.ErrUnsupported) && o.Compile {
-			o.Download = false
-			o.Printer.Logger.Info(
-				"Detected an unsupported target system, please get in touch with the Falco community. Trying to compile anyway.")
-		} else {
-			return "", fmt.Errorf("detected an unsupported target system, please get in touch with the Falco community")
-		}
+	if o.Distro.String() == driverdistro.UndeterminedDistro && o.Compile {
```

Comment:
Check is now a bit different because we are not discovering the distro here anymore; instead we receive a distro from `driver` root command.

We need to check if `ErrUnsupported` was returned, ie: if the distro is `UndeterminedDistro`.